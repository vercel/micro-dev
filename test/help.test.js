import { expect, it } from 'vitest'
import generateHelp from '../lib/help'

it('--help', () => {
  const result = generateHelp()
  expect(result).toMatchSnapshot()
})