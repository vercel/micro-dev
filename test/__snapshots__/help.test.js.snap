// Vitest Snapshot v1

exports[`--help 1`] = `
"
  Usage: micro-dev [path] [options]

  Options:

    -p, --port <n>      Port to listen on (defaults to 3000)
    -H, --host          The host on which micro will run
    -d, --dotenv        Custom path for a .env file (relative to cwd)
    -c, --cold          Disable hot reloading
    -w, --watch <dir>   A directory to watch in addition to [path]
    -L, --poll          Poll for code changes rather than using events
    -l, --limit         Size limit for JSON parsing (string like '1mb', or bytes)
    -i  --ignore <dir>  Ignore watching a file, directory, or glob
    -s, --silent        Disable requests log
    -v, --version       Output the version number
    -h, --help          Show this usage information
  "
`;
